src/Promitor.Agents.Scraper/Validation/MetricDefinitions/ResourceTypes/PowerBiDedicatedMetricValidator.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using GuardNet;
+using Promitor.Core.Scraping.Configuration.Model.Metrics;
+using Promitor.Agents.Scraper.Validation.MetricDefinitions.Interfaces;
+using Promitor.Core.Contracts.ResourceTypes;
+
+namespace Promitor.Agents.Scraper.Validation.MetricDefinitions.ResourceTypes
+{
+	internal class PowerBiDedicatedMetricValidator : IMetricValidator
+	{
```

Comment:
Nit, can you fix formatting here please?

Reply:
updated formatting, for some reason i can only see it looking weird on github, not local visual studio